modules/migration-rsync-manually-overriding.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/installing-mtc.adoc
+// * migration_toolkit_for_containers/installing-mtc-restricted.adoc
+
+:_content-type: PROCEDURE
+[id="migration-rsync-manually-overriding_{context}"]
+== Manually overriding default non-root operation for data transfer
+
+Although running Rsync pods as non-root user works in most cases, data transfer might fail when when you run workloads as root user on the source side. {mtc-short} provides two ways to manually override default non-root operation for data transfer: configuring all migrations to run an Rsync pod as root on the destination cluster for all migrations or to run an Rsync pod as root on the destination cluster per migration.
+
+In both cases, you must set the following labels on the source side of any namespaces that are running workloads with higher privileges prior to migration: `enforce`, `audit`, and `warn.`
+
+.Procedure
+
+* Prior to migration, if you are running workloads as root on the source side of a migration, set the `enforce`, `audit`, and `warn` labels on the source cluster namespaces of the workloads:
```

Comment:
Nit: since this is a procedure, it would be nice to add the step required to actually apply the label, eg., something like:

```cmd
oc label pod example-pod mylabel=myvalue
```

Reply:
Done.